src/lib/homestore_backend/hs_blob_manager.cpp

```diff
@@ -250,17 +251,23 @@ BlobManager::AsyncResult< Blob > HSHomeObject::_get_blob(ShardInfo const& shard,
         return folly::makeUnexpected(r.error());
     }
 
-    auto const blkid = r.value();
+    return _get_blob_data(repl_dev, shard.id, blob_id, req_offset, req_len, r.value() /* blkid*/);
+}
+
+BlobManager::AsyncResult< Blob > HSHomeObject::_get_blob_data(shared< homestore::ReplDev > repl_dev,
```

Comment:
This can be by `const` reference yeah? No need to copy the shared_ptr.

Reply:
Ack